### PR TITLE
OSDOCS-5334 Adding namespace option to oc adm  must-gather

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.13

Issue:
[OSDOCS-5334](https://issues.redhat.com/browse/OSDOCS-5334)

Link to docs preview:
[Preview](https://56342--docspreview.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#about-must-gather_gathering-cluster-data)

QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by cbippley** [comment]
@ingvagabund Could you take a look at this? In particular, the commands. I'm having trouble logging into a cluster to test it. Thank you!

**Comment by ingvagabund** [comment]
@cbippley looks good :)

File: modules/about-must-gather.adoc [hunk 1]
```diff
@@ -57,3 +57,11 @@ openshift-must-gather-5drcj    must-gather-s8sdh    2/2     Running     0
 ...
 ----
 // todo: table or ref module listing available images?
+Optionally, you can run the `oc adm must-gather` command in a specific namespace by using the `--run-namespace` option.
```
Comment (bergerhoffer) [inline modules/about-must-gather.adoc:60]:
```suggestion
Optionally, you can use the `--run-namespace` option to specify a namespace to run the `must-gather` tool in.
```

**Review by bergerhoffer — COMMENTED** [review]
Something like this is what I was talking about yesterday, to be more specific that you're running the tool in that namespace (and not just issuing the command against a namespace).

But honestly looking again I think your update is probably fine the way it is, so it's fine to keep it as it :)

File: modules/about-must-gather.adoc (hunk 1 above)
Comment (cbippley) [reply modules/about-must-gather.adoc:60]: Thanks!

**Comment by cbippley** [comment]
@zhouying7780 PTAL, thank you!

**Comment by lahinson** [comment]
LGTM

File: modules/about-must-gather.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/about-must-gather.adoc:60]: Are you going to implement Andrea's feedback?

**Comment by lahinson** [comment]
Hi @cbippley - This is my first rotation on the peer review squad, so another member of the squad will take a look at the PR before it goes to merge review. I'll add the `peer-review-done` label when everything is ready for merge review. Thanks!

/remove-label merge-review-needed

File: modules/about-must-gather.adoc (hunk 1 above)
Comment (cbippley) [reply modules/about-must-gather.adoc:60]: We talked about it and she said it was fine the way it was. Would you like me to change it?

**Review by lahinson — COMMENTED** [review]
This one is good to go - you can add the `merge-review-needed` label now, @cbippley :)

File: modules/about-must-gather.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/about-must-gather.adoc:60]: Yep we talked about it and I think this is good the way it is. Thanks for checking!

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**